pkg/operator/starter.go

```diff
@@ -221,7 +220,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		operatorClient,
 		controllerContext.EventRecorder,
 	).
-		WithConditionalResources(bindata.Asset, []string{"assets/alerts/kube-apiserver-slos-extended.yaml"}, notOnSingleReplicaTopology, nil).
+		WithConditionalResources(bindata.Asset, []string{"assets/alerts/kube-apiserver-slos-extended.yaml", "assets/alerts/cpu-utilization.yaml"}, notOnSingleReplicaTopology, nil).
```

Comment:
give that the `assets/alerts/cpu-utilization.yaml` alert was created on previous clusters and that now it will be created only on SNO cluster should we also delete it for clusters upgrading from prior versions ?

Reply:
Better check :) - It looks like we should be good https://github.com/openshift/cluster-kube-apiserver-operator/blob/d597184dccf293101d38e543587d40af99aca52a/vendor/github.com/openshift/library-go/pkg/operator/staticresourcecontroller/static_resource_controller.go#L167

Reply:
Could you please add a TODO where `notOnSingleReplicaTopology` is defined?

Based on our conversation from the morning the `TODO` should say that in the future the infra object might/will be changed at runtime thus the function must read the ~current state from the cluster.